api/base/serializers.py

```diff
@@ -857,20 +870,17 @@ def to_representation(self, value):
         self_url = url['self']
         self_meta = self.get_meta_information(self.self_meta, value)
         relationship = format_relationship_links(related_url, self_url, related_meta, self_meta)
```

Comment:
Investigate changes and ensure that necessary logic wasn't removed during rebasing of branch with schemablocks changes

Reply:
🆗